controllers/update_status.go

```diff
@@ -337,8 +337,9 @@ func checkAndSetProcessStatus(logger logr.Logger, r *FoundationDBClusterReconcil
 	}
 
 	processStatus := processMap[processID]
-
-	processGroupStatus.UpdateCondition(fdbv1beta2.MissingProcesses, len(processStatus) == 0)
+	// Only set the MissingProcesses condition if the machine-readable status has at least one process. We can improve this check
```

Comment:
You might want to file a radar/issue about the todo item/improvement so we won't forget about it. Thanks!